demucs/api.py

```diff
@@ -264,6 +264,26 @@ def separate_tensor(
         """
         if sr is not None and sr != self.samplerate:
             wav = convert_audio(wav, sr, self._samplerate, self._audio_channels)
+        if wav.max() == wav.min():
```

Comment:
I don't get why this is necessary ?

Reply:
If `wav.max() == wav.min()`, then `ref.std()` would be 0, which will cause L289 raise `ZeroDivisionError`

Reply:
Doing this will make it much slower
It seems no use separating an empty audio